hathitrust_api/bib_api.py

```diff
@@ -1,7 +1,11 @@
+from __future__ import absolute_import
+from __future__ import division
```

Comment:
I don't think the `division` and `print_function`  futures are needed in this file.